NSString+MD5.h

```diff
@@ -7,8 +7,25 @@
 
 @import Foundation;
 
+
+#ifdef NS_ASSUME_NONNULL_BEGIN
+#define MD5_DIGEST_ASSUME_NONNULL_BEGIN NS_ASSUME_NONNULL_BEGIN
+#else
+#define MD5_DIGEST_ASSUME_NONNULL_BEGIN
+#endif
+
+#ifdef NS_ASSUME_NONNULL_END
+#define MD5_DIGEST_ASSUME_NONNULL_END NS_ASSUME_NONNULL_END
+#else
+#define MD5_DIGEST_ASSUME_NONNULL_END
+#endif
+
+MD5_DIGEST_ASSUME_NONNULL_BEGIN
+
 @interface NSString (MD5)
 
 - (NSString *)MD5Digest;
 
 @end
+
+MD5_DIGEST_ASSUME_NONNULL_END
```

Comment:
If the above comment works can we exclude this all together? Since it's the whole file? Otherwise could we do the same `ifdef` without a new macro just down here?